docs/sql-programming-guide.md

```diff
@@ -1776,6 +1776,66 @@ working with timestamps in `pandas_udf`s to get the best performance, see
 
 ## Upgrading From Spark SQL 2.2 to 2.3
 
+  - Since Spark 2.3, Spark supports a vectorized ORC reader with a new ORC file format for ORC files and Hive ORC tables. To do that, the following configurations are newly added or change their default values.
+
+    <table class="table">
+      <tr>
+        <th>
+          <b>Property Name</b>
+        </th>
+        <th>
+          <b>Default</b>
+        </th>
+        <th>
+          <b>Meaning</b>
+        </th>
+      </tr>
+      <tr>
+        <td>
+          <code>spark.sql.orc.impl</code>
+        </td>
+        <td>
+          <code>native</code>
+        </td>
+        <td>
+          The name of ORC implementation: <code>native</code> means the native ORC support that is built on Apache ORC 1.4.1 instead of the ORC library in Hive 1.2.1. It is <code>hive</code> by default prior to Spark 2.3.
```

Comment:
I think this is newly added config. So `It is <code>hive</code> by default prior to Spark 2.3.` sounds like it is an existing config before 2.3.

Reply:
It's also updated.